documentation/content/Contributing/automated-release.md

```diff
@@ -1,42 +1,93 @@
+
+## Release Versions
+
+Version management and publishing is accomplished using the
+[semantic-release](https://github.com/semantic-release/semantic-release) tool and it's
+[configuration file](https://github.com/Pelican-Elegant/elegant/blob/master/.releaserc.json).
+This tool entirely removes any human intervention needed for determining how to label
+the next release of the project.
+
+## CHANGELOG File Updates
+
+An integral part of the release is the generation of information to be added to the
+[CHANGELOG file](https://github.com/Pelican-Elegant/elegant/blob/master/CHANGELOG.md).
+By mandating that any commits for the project follow a
+[mandated format for the commit messages]({filename}./git-commit-guidelines.md),
+the commits being added to the release can have their commit messages scanned by a tool,
+including relevant portions into the release notes. Similar to the previous paragraph on
+version management, this can be accomplished without any need for human intervention.
 
-We use [semantic-release](https://github.com/semantic-release/semantic-release) for version management and publishing.
+### Example of an Automated Release
 
-Our semantic release configuration can be viewed [here](https://github.com/Pelican-Elegant/elegant/blob/master/.releaserc.json).
+Here is an example of an automated release. This is the
+[Version 3.2.0](https://github.com/Pelican-Elegant/elegant/releases/tag/V3.2.0) that
+was released on 30-Jul-2019. The specifics about the release can be obtained by clicking
+on the 7 digit hexadecimal number below the version tag on the left side of the page, which
+will take you to the [release commit](https://github.com/Pelican-Elegant/elegant/commit/48f39643edd6c3b7449af5dae8ade6323bc7c21f).
 
-## CHANGELOG
+![automated release]({static}/images/automated-release.png)
 
-It generates the change log automatically, therefore you do not have to update the [CHANGELOG](https://github.com/Pelican-Elegant/elegant/blob/master/CHANGELOG.md).
+This release can also be viewed by performing a `git log --grep=chore(release)` command and
+looking for the specific release information in the logs. You can then view the specific
+information for that commit by using the `git show` command with the hash for the release's
+commit, `git show 48f39643edd6c3b7449af5dae8ade6323bc7c21f`.
 
-## New Releases and Version Number
+Using either process, the result will look like:
 
-semantic-release uses the commit messages to determine the type of changes in the codebase. Following [formalized conventions for commit messages]({filename}./git-commit-guidelines.md), semantic-release automatically determines the next semantic version number, generates a changelog and publishes the release.
+```text
+commit 48f39643edd6c3b7449af5dae8ade6323bc7c21f
+Author: semantic-release-bot <semantic-release-bot@martynus.net>
+Date:   Tue Jul 30 19:56:10 2019 +0000
 
-Here is an example of the release type that will be done based on a commit messages:
```

Comment:
Please re-add this table. It explains how release type is inferred by the tool and why we insist on commit format.

Reply:
@jackdewinter this feedback has not been addressed